cleverhans/picklable_model.py

```diff
@@ -562,10 +563,22 @@ def get_params(self):
   def fprop(self, x, **kwargs):
     # TODO: before adding dataset augmentation, we didn't have to do this.
```

Comment:
I believe we can remove the TODO if we removed the with tf.device() call. Is this correct @goodfeli ?